### ISSUE TITLE
Is there pretrained versions for those models?

### ISSUE THREAD

**Issue by anpc849** [ArieSeirack/DHVT#3]
Where i can find the pretrained models?
Thanks for helping

**Comment by ArieSeirack** [comment]
> Where i can find the pretrained models? Thanks for helping

yes, the pretrained models are coming along with the next version in a few days

**Comment by dessyfitriaya** [comment]
> > Where i can find the pretrained models? Thanks for helping
> 
> yes, the pretrained models are coming along with the next version in a few days

have you uploaded the pretrained models? i cant find it here sorry
